sda-commons-server-kafka/src/main/java/org/sdase/commons/server/kafka/KafkaBundle.java

```diff
@@ -383,19 +457,88 @@ private ExpectedTopicConfiguration createTopicDescription(TopicConfig c) {
     return new KafkaConsumer<>(consumerProperties, keyDeSerializer, valueDeSerializer);
   }
 
+  /**
+   * Creates and registers a message producer with a String key serializer and the default producer
+   * configuration
+   */
+  public <V> MessageProducer<String, V> createAndRegisterProducer(
+      Class<V> valueClass, ObjectMapper objectMapper, String topicName) {
+    return createAndRegisterProducer(
+        String.class, valueClass, objectMapper, topicName);
+  }
+
+  /**
+   * Creates and registers a message producer with a default producer configuration
+   */
+  @SuppressWarnings("unchecked")
+  public <K, V> MessageProducer<K, V> createAndRegisterProducer(
+      Class<K> keyClass, Class<V> valueClass, ObjectMapper objectMapper, String topicName) {
+
+    ProducerRegistration<K, V> producerRegistration =
+        ProducerRegistration.<K, V>builder()
```

Comment:
```suggestion
        ProducerRegistration.builder()
```

When I remember the changes we did for DW2 correctly, this shouldn't be necessary anymore.